phpdotnet/phd/Format/Abstract/PDF.php

```diff
@@ -132,7 +132,8 @@ class PdfWriter {
     // To temporarily store $current(s)
     private $old = array();
 
-    function __construct($pageWidth = 210, $pageHeight = 297) {
+    function __construct($pageWidth = 210, $pageHeight = 297, Config $config) {
```

Comment:
Deprecated: Optional parameter $a declared before required parameter $b is implicitly treated as a required parameter

Reply:
You're right. What do you think about moving `$pageWidth` and `$pageHeight` to `Config`? These are configuration details so should probably be properties of `Config`, right?

Reply:
Yes or you could make the Cofig as the first parameter

Reply:
I've just realized that `PdfWriter` is not even a `Format` so doesn't need an instance of `Config` at all. I'm removing it from the parameter list and removing the next line as well.